internal/domain/smtp.go

```diff
@@ -5,5 +5,10 @@ type SMTPConfigState int32
 const (
 	SMTPConfigStateUnspecified SMTPConfigState = iota
 	SMTPConfigStateActive
+	SMTPConfigStateInactive
```

Comment:
just noted by coincidence: this change would effect existing smtp configuration, which were removed. they would be considered inactive instead

Reply:
Thanks @livio-a, I've moved the new inactive state after existing removed state. Whenever I play with iota in the future I'll be more careful :sweat_smile: